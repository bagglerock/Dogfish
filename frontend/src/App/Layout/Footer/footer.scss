.footer {
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  color: white;
  text-align: center;
  background-color: darkolivegreen;
  height: 50px;
  border-top: solid white 2px;
  font-size: 10px;

  h1 {
    color: white;
  }
}
