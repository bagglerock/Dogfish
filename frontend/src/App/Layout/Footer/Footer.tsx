import React from 'react';
import './footer.scss';

export const Footer: React.FC = () => {
  const date = new Date(Date.now());
  const year = date.getFullYear();

  return (
    <footer className="footer pt-3">
      <h1>&copy; {year} Oscar Villalta</h1>
    </footer>
  );
};
