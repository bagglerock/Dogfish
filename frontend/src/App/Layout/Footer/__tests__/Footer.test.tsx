import { render } from '@testing-library/react';
import { Footer } from 'app/Layout/Footer/Footer';

test('renders App/Layout', () => {
  Date.now = jest.fn(() => new Date('2000-01-01T01:00:00Z').getTime());

  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
