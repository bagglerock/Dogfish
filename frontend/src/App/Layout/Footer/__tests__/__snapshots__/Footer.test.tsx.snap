// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App/Layout 1`] = `
<div>
  <footer
    class="footer pt-3"
  >
    <h1>
      Â© 
      1999
       Oscar Villalta
    </h1>
  </footer>
</div>
`;
